apps/tup-cms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM taccwma/core-cms:831f541
+FROM taccwma/core-cms:2581ba6
```

Comment:
‼️ This is a Core-CMS—off—`main` image. Once https://github.com/TACC/Core-CMS/pull/604 is merged, and `dev/tup-cms` updated, build and save an image off of `dev/tup-cms`. ‼️

Reply:
Hotfixed in https://github.com/TACC/tup-ui/pull/161.